Caio_Replica/src/json/JSONReader.java

```diff
@@ -0,0 +1,44 @@
+package json;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+public class JSONReader {
+
+	public static void main(String[] args) {

```

Comment:
Does it need to be it's own process (main)? I think it would make more sense that the RM could just create an instance of the JSONReader class, call some Init() function which would read from the config file and instantiate some member variable dictionary with all the mappings, then call a public method like int getPortForKeys(string key, string key2) and these keys would be like key1: "Mark", key2: "MTL". Or just make 1 key like "Mark-MTL".

Right now I'm not sure how the RM would use the class to retrieve the ports. Let me know if I'm on the same page.